internal/apis/certmanager/validation/certificate_test.go

```diff
@@ -552,14 +552,14 @@ func TestValidateCertificate(t *testing.T) {
 		"invalid certificate with incorrect email": {
 			cfg: &internalcmapi.Certificate{
 				Spec: internalcmapi.CertificateSpec{
-					EmailAddresses: []string{"aliceexample.com"},
+					EmailAddresses: []string{"alice.example.com"},
```

Comment:
domains shouldn't be randomly minted. [RFC 2606](https://www.rfc-editor.org/rfc/rfc2606.html) is available for use, so...

Note that I didn't fix all instances of that, but this one seemed safe enough.